docs/crypto.md

```diff
@@ -1,17 +1,17 @@
-# Bitcoin Crypto
-The cryptographic primitives (ECDSA and HMAC) implementations in this package have been reviewed by the BitPay engineering team. More audits and reviews are welcomed.
+# Dash Crypto
+The cryptographic primitives (ECDSA and HMAC) implementations in this package have been reviewed by the BBitPay engineering team. More audits and reviews are welcomed.
```

Comment:
Typo `BBitPay`. Also, not sure if we can state that Dash Crypto was reviewed by them. Probably should say that upstream was reviewed and this fork comes with no changes or smth like that.